routing/index_graph_starter_joints.cpp

```diff
@@ -330,7 +336,8 @@ std::vector<JointEdge> IndexGraphStarterJoints::FindFirstJoints(Segment const &
     Segment segment = queue.front();
     queue.pop();
     Segment beforeConvert = segment;
-    // Либо сегмент фейковый и из него получается реальный, который joint, либо он настоящий и joint.
+    // Either segment is fake and it converts to real, the end RoadPoint of which if Joint,
```

Comment:
конечная точка которого является джоинтом
Either the segment is fake and it can be converted to real one with Joint end (RoadPoint),
or it's the real one and its end (RoadPoint) is Joint.